### PR TITLE
Add `FLUSHALL` command.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
https://redis.io/docs/latest/commands/flushall/

Tested manually.
IT disabled for this command, because it could affect other tests. It also blocks option to investiage failures of other tests, because it clears the DB.
Will reactivate IT after implementing fix for [51963289](https://github.com/orgs/Bit-Quill/projects/4/views/5?filterQuery=redis&pane=issue&itemId=51963289).

### CODE REVIEW

File: java/client/src/main/java/glide/api/models/commands/FlushAllOption.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import glide.api.models.configuration.RequestRoutingConfiguration.Route;
+
+/**
+ * Defines flushing mode for <code>FLUSHALL</code> command implemented by {@link
+ * RedisClient#flushall(FlushAllOption)}, {@link RedisClusterClient#flushall(FlushAllOption)}, and
+ * {@link RedisClusterClient#flushall(FlushAllOption, Route)}.
+ *
+ * @see <a href="https://redis.io/commands/flushall/">redis.io</a>
+ */
+public enum FlushAllOption {
```
Comment (jonathanl-bq) [inline java/client/src/main/java/glide/api/models/commands/FlushAllOption.java:15]: Do we want to call it FlushAllOption when FLUSHDB also uses the same option?

**Review by jonathanl-bq — APPROVED** [review]
Conditional approval on Andrew's comment about creating a new testing framework for this.

File: java/client/src/main/java/glide/api/models/commands/FlushAllOption.java (hunk 1 above)
Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/models/commands/FlushAllOption.java]: Good catch, renamed

**Comment by Yury-Fridlyand** [comment]
Options for IT:
1. Implement controlling of redis instances in IT from java code to spin up a new cluster for that test (see task [51963289](https://github.com/orgs/Bit-Quill/projects/4/views/5?filterQuery=redis&pane=issue&itemId=51963289))
2. Add another test suite(s)
3. Do not test this in IT
4. Keep as is
5. Keep as is until we do 1
6. Do not test until we do 1

Option 5 is my favorite

**Review by jonathanl-bq — APPROVED** [review]

**Comment by Yury-Fridlyand** [comment]
Re: IT
IT moved to another sub-suite and disabled until we complete [51963289](https://github.com/orgs/Bit-Quill/projects/4/views/5?filterQuery=redis&pane=issue&itemId=51963289)

**Pull request merged**